Compile/src/Producers/LeptonSFProducer.cc

```diff
@@ -120,17 +120,19 @@ void LeptonIDSFProducer::Produce(ZJetEvent const& event,
                                  ZJetSettings const& settings) const
 {
     if(product.m_zValid){
-        product.m_weights["zl1IDSFWeight"] = 1/GetScaleFactor(0, *product.m_zLeptons.first);
-        product.m_weights["zl2IDSFWeight"] = 1/GetScaleFactor(0, *product.m_zLeptons.second);
+        const auto* lep1 = product.m_originalLeptons.at(product.m_zLeptons.first);
```

Comment:
Just for my understanding:
Is there a reason for using the m_originalLeptons?
This should be fine but we in principle can also use the first two m_validLeptons, or am I wrong?